packaging/datadog-agent/source/setup_agent.sh

```diff
@@ -447,11 +447,19 @@ then
   INTEGRATIONS=$(ls $DD_HOME/integrations/)
   for INT in $INTEGRATIONS; do
     INT_DIR="$DD_HOME/integrations/$INT"
+    pushd $INT_DIR
```

Comment:
since you're using `pushd` here you can remove all the `$INT_DIR` prefixes to the paths you use below (unless I'm missing something...)

Reply:
It looks much cleaner now!